commands.go

```diff
@@ -31,6 +34,7 @@ func CmdAgent(ctx *cli.Context) {
 	})
 }
 
+
```

Comment:
please remove this unnecessary extra whitespace

Reply:
Done  f46ae53